### PR TITLE
Display tagline

### PR DESCRIPTION
**Pull request by sssoz**
Update tagline

See DOAJ/doajPM#3153

### CODE REVIEW

File: portality/templates/includes/footer.html [hunk 1]
```diff
@@ -15,8 +15,15 @@ <h2 class="sr-only">
...
+                <a href="/" class="logo">
+                    <h2 class="sr-only">The Directory of Open Access Journals</h2>
+                    {% include "includes/svg/logo.svg" %}
+                    <span class="logo__tagline">
+                      Open. <br/>Global. <br/>Trusted.
+                    </span>
+                </a>
                 <p>
-                    © DOAJ 2020 default by all rights reserved <a href="{{ url_for('doaj.terms') }}" style="text-decoration: underline;">unless otherwise specified</a>.
+                    © DOAJ 2022 default by all rights reserved <a href="{{ url_for('doaj.terms') }}" style="text-decoration: underline;">unless otherwise specified</a>.
```
Comment (sssoz) [inline portality/templates/includes/footer.html:26]: @richard-jones This isn’t ideal of course, but couldn’t figure out how to get the current year in Jinja templates

Comment (richard-jones) [reply portality/templates/includes/footer.html:26]: Apparently we used to have this and we must have just lost it during the redesign.  If you just do {{ current_year }} it'll give you the current year

Comment (sssoz) [reply portality/templates/includes/footer.html]: Thank you! It works — committed.

**Review by richard-jones — APPROVED** [review]

**Review by Steven-Eardley — APPROVED** [review]

**Comment by Steven-Eardley** [comment]
Not sure why this PR is still open, code is merged.

**Pull request closed without merging**